src/plays/tube2tunes/Tube2tunes.js

```diff
@@ -7,7 +7,7 @@ import Loader from './components/Loader';
 import TubeTunesLogo from './components/tubeTuneLogo.svg';
 
 // WARNING: Do not change the entry componenet name
-function Tube2tunes(props) {
+function Tube2Tunes(props) {
```

Comment:
Hi @WhyNotSantosh , I think we need to either rename this file to `Tube2Tunes.js` or change the export in `src/plays/index.js` file from `export { default as Tube2Tunes } from 'plays/tube2tunes/Tube2Tunes';` to `export { default as Tube2Tunes } from 'plays/tube2tunes/Tube2tunes';`.
I have tried to fix this same issue in my below PR as well. But if you want to fix this issue in this PR instead, then I can remove it from my PR.
https://github.com/reactplay/react-play/pull/1088